src/main/java/org/zalando/nakadi/filters/LoggingFilter.java

```diff
@@ -55,7 +56,7 @@ protected void doFilterInternal(final HttpServletRequest request,
                     .orElse("-");
             final Long contentLength = request.getContentLengthLong() == -1 ? 0 : request.getContentLengthLong();
 
-            LOG.info("[ACCESS_LOG] {} \"{}{}\" \"{}\" \"{}\" statusCode: {} {} ms \"{}\" \"{}\" {} bytes",
+            ACCESS_LOGGER.info("[ACCESS_LOG] {} \"{}{}\" \"{}\" \"{}\" statusCode: {} {} ms \"{}\" \"{}\" {} bytes",
```

Comment:
@antban 
1) I think another quick win will be to make `[ACCESS_LOG]` string shorter
2) and also we can remove "statusCode:" string as our parser extract the actual value anyway
3) we can also replace `bytes` with 'B'